docs/ConnectPOB/manage-nodes.md

```diff
@@ -83,3 +140,116 @@ When clicking a node item this extended information that is attached to the node
+|     D     | Is the battery charging. True = yes, False = no.                                                                                                                                                                   |
+|     E     | The temperature the node is registering at its placed location.                                                                                                                                                    |
+|     F     | The voltage the node is receiving from the socket it is connected to. If this is 0, then the node is either disconnected from a power socet or the power socet is not functioning correctly (power outage/defect). |
+|     G     | Uptime??                                                                                                                                                                                                           |
+|     H     | Signal strength to parent node in desibel. This value will alway be minus.                                                                                                                                         |
+|     I     | What version is the firmware in the node running. This should be the same as Node data 2.                                                                                                                          |
+|     J     | What BLE (Bluetooth Low Energy) version is the node running. This should be the same as Node data 2.                                                                                                               |
+|     K     | Nordic out of sec ......                                                                                                                                                                                           |
+|     L     | Buffers full-count.....                                                                                                                                                                                            |
+|     M     | Queue full-count ......                                                                                                                                                                                            |
+|     N     | Lowest Number of buffers available........                                                                                                                                                                         |
+|     O     | Nordic CRC error count......                                                                                                                                                                                       |
+
+#### MDT - Node data 2
+
+This is the chip in the node that handles .......
+![Mesh Debug Tool node data section 2](node-admin-images/mesh-debug-images/mesh-debug-tool-part-3.png)
```

Comment:
```suggestion
```
Same here, not really needed as we already have a screenshot of the full debug view above?